packages/plugin-infra/src/types.ts

```diff
@@ -412,7 +412,7 @@ export namespace Plugin.SNSAdaptor {
             /**
              * If it returns false, this tab will not be displayed.
              */
-            shouldDisplay?: (identity?: ProfileIdentity, addressNames?: ProfileAddress[]) => boolean
+            shouldDisplay?(identity?: ProfileIdentity, addressNames?: ProfileAddress[], extraInfo?: any): boolean
```

Comment:
```suggestion
            shouldDisplay?(identity?: ProfileIdentity, addressNames?: ProfileAddress[], extraInfo?: { type: 'daoTabTwitterIdList', items: string[] }): boolean
```

In this way, it can be a bit platform nature because we can add more discriminated unions onto it.

Reply:
done